### PR TITLE
Fixed error handling and made it to where users can cancel out of afaf

### PR DESCRIPTION
**Pull request by jonathan-elize**

### CODE REVIEW

**Review by DarrylBrooks97 — APPROVED** [review]
LGTM

File: functions/listeners/config.js [hunk 1]
```diff
@@ -9,4 +9,3 @@ discordClient.login(access_token).catch((err) => console.log(err));
 exports.discordClient = discordClient;
 exports.discordGuilds = discordClient.guilds.cache;
 exports.discordChannels = discordClient.channels.cache;
```
Comment (Garrett1Tolbert) [inline functions/listeners/config.js:11]: @jonathan-elize add back the discordMessageEmbed. Why did you remove it

File: functions/listeners/afaf/afaf.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
+const { MessageEmbed } = require('discord.js');
```
Comment (Garrett1Tolbert) [inline functions/listeners/afaf/afaf.js:1]: Remove this and import discordMessageEmbed from the listeners config

File: functions/listeners/afaf/afaf.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
 const { blaccLogo, blaccSmithServer } = require('../../constants');
-const { discordGuilds, discordChannels } = require('../config');
 
-exports.handleAFAF = async (message) => {
+exports.handleAFAF = async ({discordClient, discordGuilds, message}) => {
```
Comment (Garrett1Tolbert) [inline functions/listeners/afaf/afaf.js:4]: The only thing that needs to be passed here is the message. Import discordClient and discordGuilds from the listeners config

File: functions/listeners/afaf/afaf.js [hunk 4]
```diff
@@ -15,51 +14,65 @@ exports.handleAFAF = async (message) => {
...
+		}
+	}
+	catch(err){
+		throw({getPreferredChannel: err})
+	}
+};
 
 const embedMessage = (question) => {
-	return discordMessageEmbed
+	return new MessageEmbed()
```
Comment (Garrett1Tolbert) [inline functions/listeners/afaf/afaf.js:75]: use discordMessageEmbed instead

File: functions/listeners/discordListener.js [hunk 5]
```diff
@@ -1,12 +1,12 @@
 const { handleAFAF } = require('./afaf');
-const { discordClient } = require('./config');
+const { discordClient, discordGuilds } = require('./config');
```
Comment (Garrett1Tolbert) [inline functions/listeners/discordListener.js:2]: you don't need to import discordGuilds here

File: functions/listeners/discordListener.js [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 
 module.exports = () => {
 	discordClient.on('message', async (message) => {
 		try {
-			await handleAFAF(message);
+			await handleAFAF({discordClient, discordGuilds, message});
```
Comment (Garrett1Tolbert) [inline functions/listeners/discordListener.js:7]: you only need to pass the message to this function

**Review by Garrett1Tolbert — CHANGES_REQUESTED** [review]
